docs/qs-editwiki.md

```diff
@@ -0,0 +1,103 @@
+# How to edit SuperMemo.wiki
+
+### Requirements
+
+#### Validate with us the changes you want to apply
+
+Make sure that you either:
+- Edit the wiki to address an issue that has already been accepted,
+- Open an issue **beforehand** to validate with us the nature and the reasons for the changes you want to apply.
+
+#### Technical requirements
+
+The minimum requirement is an account on [GitHub](https://github.com/) in order to send us your changes to the wiki.
+
+### Setup
+
+#### Forking the Documentation
+
+After logging in to your GitHub account:
+1. Visit the [official wiki repository](https://github.com/supermemo/Documentation). This repository contains the documentation for both [SuperMemo](https://www.supermemo.wiki) and [SuperMemo Assistant](https://www.supermemo.wiki/sma).
+2. Click on the Fork button at the top right of your screen. This will create create a new, personal version of the repository on your account.
+
+![Fork Button](content/images/editwiki/forkbutton.png)
+
+> [!NOTE]
+> Your repository should be available at github.com/**YourUsername**/Documentation/, provided you didn't change the name of the repository.
+
+#### Setting up Docsify on your computer
+
+If your edit is simple and less than a few paragraphs worth, you may skip this step. In this case, go to [Editing on GitHub](#on-github).
+
+⚠️ This section is a work in progress.
+
+### Editing the wiki
+
+#### On GitHub
+
+##### Editing the wiki on GitHub
+
+!> Only edit on GitHub if your edit **really is** minor. For larger modifications, we require that you set up a local version of the documentation on your computer. This enables you to validate the visual result of your changes on the website, in real-time.
+
+1. On GitHub, navigate to your forked repository, and select the `docs` folder. 
+
+![Docs Folder](content/images/editwiki/docsfolder.png)
+
+2. Find the file you want to edit. For example, if you are looking to edit the installation page, you would select the `qs-installation.md` page.
+
+> [!TIP]

```

Comment:
You can use this format 
```
> [!NOTE]
> Your text goes here
```

instead of `> HINT!`.

This will create a coloured rectangle that puts your text into context.
https://github.com/zanfab/docsify-plugin-flexible-alerts

Try it next time and see what looks best.